src/sentry/static/sentry/app/components/organizations/projectSelector/index.tsx

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import sortBy from 'lodash/sortBy';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import DropdownAutoComplete from 'app/components/dropdownAutoComplete';
+import Button from 'app/components/button';
+import theme from 'app/utils/theme';
+import {IconAdd} from 'app/icons';
+import {GetActorPropsFn} from 'app/components/dropdownMenu';
+
+import SelectorItem from './selectorItem';
+
+const defaultProps = {
+  // Allow selecting multiple projects
+  multi: false,
+};
+
+//TODO(ts): replace the types below with the DropdownAutoComplete type
+type Actions = {
+  open: () => void;
+  close: () => void;
+};
+type ChildrenProps = {
+  actions: Actions;
+  getActorProps: GetActorPropsFn;
+  getInputProps: Function;
+  isOpen: boolean;
+  selectedProjects: Project | null;
+};
+
+type MenuFooterProps = {
+  actions: Actions;
+};
+
+type Props = {
+  organization: Organization;
+  // used by multiProjectSelector
+  multiProjects: Array<Project>;
```

Comment:
the old code supported also `string[]` but I thought this was wrong as after testing it using different organizations I could not find any use case. I'll add it back just in case ..thanks for pointing that out :)

Reply:
@markstory I checked some use cases from large companies and I couldn't find any that passed an array of strings ... so I believe that it's fine